src/Viewer3D/CityMapGeometry.cpp

```diff
@@ -0,0 +1,73 @@
+#include "CityMapGeometry.h"
+
+CityMapGeometry::CityMapGeometry()
+{
+    m_bld_map_reader = nullptr;
+    m_model_name = "city_map_defualt_name";
+    vertexData.clear();
+    map_loaded_flag = 0;
+
+    main_timer = new QTimer(this);
+    main_timer->start(500);
+    connect(main_timer, &QTimer::timeout, this, &CityMapGeometry::mainTimerEvent);
+
+    updateData();
+}
+
+void CityMapGeometry::setModelName(QString m_name)
+{
+    m_model_name = m_name;
+    //    setName(m_model_name);
+
+    emit modelNameChanged();
+}
+
+void CityMapGeometry::setCityMap(QString map_path)
+{
+    if(m_city_map_path.compare(map_path) == 0)
```

Comment:
Even single line ifs should have bracing. Single line ifs have a tendency to turn into multi-line down the road. Which can then lead to bugs when people don't noticed they need to add bracing.